src/ReactImageMagnify.js

```diff
@@ -21,14 +21,31 @@ const ReactImageMagnify = ({
     largeImage,
     lensStyle,
     smallImage,
-    style
+    style,
+    enlargedImagePosition
 }) => {
 
     const cursorOffset = {
         x: Math.round(((smallImage.width / largeImage.width) * smallImage.width) / 2),
         y: Math.round(((smallImage.height / largeImage.height) * smallImage.height) / 2)
     };
-    const defaultLensStyle = { backgroundColor: 'rgba(0,0,0,.4)' };
+    
+    let defaultLensStyle = {};
+    
+    switch (enlargedImagePosition) {
```

Comment:
What do you think about changing this to something like the following? It seems simpler to me.
```
    let defaultLensStyle = {
        backgroundColor: 'rgba(0,0,0,.4)'
    };
    
    if (enlargedImagePosition === 'over') {
        defaultLensStyle = {};
    }
```